e2e/tests/features/journeys/registration/registrationStage1Test.e2e.ts

```diff
@@ -0,0 +1,159 @@
+import { test } from "../../../../hooks/CustomFixtures";
+import { assertErrorUtil } from "../../../../utils/errorUtils";
+
+// --- Success TESTS --- //
+test.describe("Successful user registration journey", () => {
+  test("Successful registration journey", async ({
+    namePage,
+    dateOfBirthPage,
+    emailPage,
+    passwordPolicyPage,
+    consentPage,
+    emailVerificationPage,
+  }) => {
+    await test.step("Given I have navigated to the name page", async () => {
+      await namePage.goTo();
+    });
+    await test.step("When I see the components correctly rendered", async () => {
+      await namePage.assertTextVisible();
+      await namePage.assertButtonsVisible();
+      await namePage.assertFormVisible();
+      await namePage.assertCorrectContent();
+    });
+    await test.step("Then I will not see any error messages rendered", async () => {
+      await namePage.assertErrorsHidden();
+    });
+    await test.step("When I fill only the first name input", async () => {
+      await namePage.firstNameInput.fill("firstName");
+    });
+    await test.step("When I press continue", async () => {
+      await namePage.continueButton.click();
+    });
+    await test.step("I will see an error message correctly rendered", async () => {
+      await assertErrorUtil(
```

Comment:
Would prefer you to use a page method here, which calls the assertErrorUtil method
You could even pass the expected error message in from here if you like.
E.g. 
await namePage.assertLastNameError("Error: Enter your last name")

And on NamePage file:
async assertLastNameError(message: string) {
    await assertComponentWithText(this.lastNameError, message)
  }